<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TeacherGo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>651</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>New Course</string>
     </attribute>
     <widget class="QLineEdit" name="classNameBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="majorBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Computer Science</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="sectionBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>120</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lastNameBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>170</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="firstNameBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>200</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>59</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Class</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>59</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Major</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>59</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Section #</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>18</x>
        <y>170</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Last name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>18</x>
        <y>200</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>First name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>59</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Professor</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="startGroupBox">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>50</y>
        <width>151</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Start Time</string>
      </property>
      <widget class="QTimeEdit" name="startTime">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>118</width>
         <height>23</height>
        </rect>
       </property>
       <property name="maximumTime">
        <time>
         <hour>23</hour>
         <minute>59</minute>
         <second>59</second>
        </time>
       </property>
       <property name="minimumTime">
        <time>
         <hour>7</hour>
         <minute>0</minute>
         <second>0</second>
        </time>
       </property>
       <property name="displayFormat">
        <string>HH:mm</string>
       </property>
       <property name="time">
        <time>
         <hour>7</hour>
         <minute>0</minute>
         <second>0</second>
        </time>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="endGroupBox">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>150</y>
        <width>151</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>End Time</string>
      </property>
      <widget class="QTimeEdit" name="endTime">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>118</width>
         <height>23</height>
        </rect>
       </property>
       <property name="minimumTime">
        <time>
         <hour>7</hour>
         <minute>0</minute>
         <second>0</second>
        </time>
       </property>
       <property name="displayFormat">
        <string>HH:mm</string>
       </property>
       <property name="time">
        <time>
         <hour>8</hour>
         <minute>0</minute>
         <second>0</second>
        </time>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="dayGroup">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>250</y>
        <width>151</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>Day(s)</string>
      </property>
      <widget class="QCheckBox" name="monBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>85</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Monday</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="friBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>85</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Friday</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="thurBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>85</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Thursday</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="wedBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Wednesday</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="tuesBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>85</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Tuesday</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="termGroup">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>171</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>Term Length</string>
      </property>
      <widget class="QRadioButton" name="termRadio">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>100</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Full term</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="firHTermRadio">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Half-term(first half)</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="sndHTermRadio">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Half-term(seond half)</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>410</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Add section</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>List of Courses</string>
     </attribute>
     <widget class="QTableWidget" name="sectTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>611</width>
        <height>391</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>Class</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Major</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Section</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Professor</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Day &amp; Time</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Term</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>663</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTeacherGo">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuTeacherGo"/>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
